__tests__/app.test.js

```diff
@@ -35,4 +35,38 @@ describe('app', () => {
         });
     });
   });
+  describe('GET - /api/articles/:article_id', () => {
+    test('Status: 200 - should respond with specified article in an object', () => {
+      return request(app)
+        .get('/api/articles/1')
+        .expect(200)
+        .then(({ body }) => {
+          expect.objectContaining({
```

Comment:
Shouldn't the single article live in a key of article instead of being free in the body?